docs/spark-procedures.md

```diff
@@ -283,11 +283,27 @@ Iceberg can compact data files in parallel using Spark with the `rewriteDataFile
 | `options`     | ️   | map<string, string> | Options to be used for actions|
 | `where`       | ️   | string | predicate as a string used for filtering the files. Note that all files that may contain data matching the filter will be selected for rewriting|
 
+#### Options
+
+| Name | Default Value | Description |
+|------|---------------|-------------|
+| `max-concurrent-file-group-rewrites` | 5 | Maximum number of file groups to be simultaneously rewritten |
```

Comment:
This is great, wondering, can we take the text directly from the javadoc?  Looks like that has more information.

Reply:
Do you mean copy all the javadoc comments? Will that be too long? I'm trying to put a summary here.

Reply:
Yea took a second look, you are right, some of them are too long.  

But now we are removing references to javadoc, I suggest to still add back more context, I added suggestion to individual options.